internal/api/schemas/v1.yaml

```diff
@@ -11,17 +11,168 @@ info:
     name: Mozilla Public License 2.0
     url: https://www.mozilla.org/en-US/MPL/2.0/
 paths:
+  "/namespaces/{namespace}/gateways":
```

Comment:
We'll need a `POST` on this path in order to create any gateways outside of the default namespace.

I think we'd just want `POST /gateways` to be an alias for `POST /namespaces/default/gateways`